src/services/indexer/indexer.ts

```diff
@@ -245,4 +252,58 @@ export class Indexer {
       throw err;
     }
   }
+
+  async sushiPools(): Promise<SushiPoolResponse> {
+    try {
+      if (!app.provider.indexer) {
+        throw new Error('Indexer is not configured');
+      }
+
+      const response = await jsonrpcRequest<SushiPoolResponse>(
+        app.provider.indexer,
+        'sushiPools',
+        [],
+      );
+
+      return response || {};
+    } catch (err) {
+      if (err instanceof JSONRPCError) {
+        this.captureException(err, 'Indexer:sushiPools', err.meta);
+      }
+      throw err;
+    }
+  }
+
+  async sushiPoolEstimate({
+    amount,
+    sender,
+    token_in,
+    token_out,
+    currency_id = Currencies.currency?.id,
```

Comment:
sushiPoolEstimate is not an action and Indexer is not an observer, so `Currencies.currency` may have not relevant value - keep it in mind please